lib/game.rb

```diff
@@ -26,7 +58,7 @@ def self.games=(value)
 
   def initialize(data)
     @game_id = data[:game_id]
-    @season = data[:season]
+    @season = data[:season].to_s
```

Comment:
Why did you need to turn this into a string?

Reply:
I think whenever the spec harness asks for a season as a return value it asks for the `season` to be represented as a `String` for example;

```
{
"20122013" => 3.86,
"20162017" => 4.75,
"20142015" => 4.00,
"20152016" => 3.88,
"20132014" => 4.33
}